README.md

```diff
@@ -25,6 +25,17 @@ The power is at your fingertips.
 - Get up and running in seconds within GitHub and GitLab with the available [action](https://github.com/purpleclay/nsv-action) or [template](https://gitlab.com/purpleclay/nsv).
 - First class [Dagger](https://dagger.io/) support.
 
+## Installation
+
+### Arch Linux
+
+[nsv](https://aur.archlinux.org/packages/nsv) is available as a package on the AUR.
+You can install it using an AUR helper (e.g. `yay`):
+
+```sh
+yay -S nsv
+```
```

Comment:
Awesome. Would you be able to make this edit to this markdown file instead: https://github.com/purpleclay/nsv/blob/main/docs/install/binary.md?plain=1#L30

I have just checked my `.goreleaser.yaml` file and I have clearly lied in the `aur` section in the docs 😊 . Will your aur package stay in sync with new releases automatically?

Reply:
Great. Approving. And nice tool btw.

Reply:
thanks